src/composables/fungibleTokens.ts

```diff
@@ -50,6 +50,10 @@ const { onNetworkChange } = createNetworkWatcher();
 const availableTokensPooling = createPollingBasedOnMountedComponents(60000);
 const tokenBalancesPooling = createPollingBasedOnMountedComponents(10000);
 
+/**
+ * AE network AEX-9 fungible tokens
+ * TODO With the addition of ERC-20 tokens we should rename this to useAex9Tokens
```

Comment:
I don't full;y understand this comment. I think this composable should take care of all tokens - no matter if ERC-20 or Aex-9.